spec/requests/projects_spec.rb

```diff
@@ -117,6 +119,25 @@
     end
   end
 
+  # Test suite for PUT /project/:project_id/sprint-planning-complete/:sprint_id
+  describe 'PUT /project/:project_id/sprint-planning-complete/:sprint_id' do
+    let(:completed_sprint_id) { sprints.first.id }
+
+    before { put "/projects/#{id}/sprint-planning-complete/#{completed_sprint_id}" }
+
+    context 'when the sprint planning exists' do
+      it 'updates the record' do
+        print(json)
+        expect(json['planned_sprint_ids']).to eq([completed_sprint_id])
```

Comment:
this not, here it says there is not sprint like that.
But all sprint in the course should be listed.